<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1300</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1300</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1300</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Result</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget{
background-color: rgb(0, 0, 0);
}
QToolTip{
border:1px solid rgb(0, 0, 0);
background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);
font: 10pt &quot;Microsoft YaHei&quot;;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="topologyview" native="true">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>100</y>
      <width>1000</width>
      <height>700</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-left: 1px solid #dedede;</string>
    </property>
   </widget>
   <widget class="QWidget" name="systab" native="true">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>0</y>
      <width>1000</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-bottom: 1px solid #dedede;</string>
    </property>
    <widget class="QPushButton" name="sys1">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>15</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-bottom-width: 0px;
	background-color: rgb(0, 170, 255);
	border-radius: 35px;
	font: 12pt &quot;Microsoft YaHei&quot;;
	border-style: outset;
}
QPushButton:hover{
	background-color:rgb(0, 147, 220)
}</string>
     </property>
     <property name="text">
      <string>1</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sys2">
     <property name="geometry">
      <rect>
       <x>115</x>
       <y>15</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-bottom-width: 0px;
	background-color: rgb(0, 170, 255);
	border-radius: 35px;
	font: 12pt &quot;Microsoft YaHei&quot;;
	border-style: outset;
}
QPushButton:hover{
	background-color:rgb(0, 147, 220)
}</string>
     </property>
     <property name="text">
      <string>2</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sys3">
     <property name="geometry">
      <rect>
       <x>215</x>
       <y>15</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-bottom-width: 0px;
	background-color: rgb(0, 170, 255);
	border-radius: 35px;
	font: 12pt &quot;Microsoft YaHei&quot;;
	border-style: outset;
}
QPushButton:hover{
	background-color:rgb(0, 147, 220)
}</string>
     </property>
     <property name="text">
      <string>3</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sys4">
     <property name="geometry">
      <rect>
       <x>315</x>
       <y>15</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-bottom-width: 0px;
	background-color: rgb(0, 170, 255);
	border-radius: 35px;
	font: 12pt &quot;Microsoft YaHei&quot;;
	border-style: outset;
}
QPushButton:hover{
	background-color:rgb(0, 147, 220)
}</string>
     </property>
     <property name="text">
      <string>4</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sys5">
     <property name="geometry">
      <rect>
       <x>415</x>
       <y>15</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-bottom-width: 0px;
	background-color: rgb(0, 170, 255);
	border-radius: 35px;
	font: 12pt &quot;Microsoft YaHei&quot;;
	border-style: outset;
}
QPushButton:hover{
	background-color:rgb(0, 147, 220)
}</string>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sys6">
     <property name="geometry">
      <rect>
       <x>515</x>
       <y>15</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-bottom-width: 0px;
	background-color: rgb(0, 170, 255);
	border-radius: 35px;
	font: 12pt &quot;Microsoft YaHei&quot;;
	border-style: outset;
}
QPushButton:hover{
	background-color:rgb(0, 147, 220)
}</string>
     </property>
     <property name="text">
      <string>6</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sys7">
     <property name="geometry">
      <rect>
       <x>615</x>
       <y>15</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-bottom-width: 0px;
	background-color: rgb(0, 170, 255);
	border-radius: 35px;
	font: 12pt &quot;Microsoft YaHei&quot;;
	border-style: outset;
}
QPushButton:hover{
	background-color:rgb(0, 147, 220)
}</string>
     </property>
     <property name="text">
      <string>7</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sys8">
     <property name="geometry">
      <rect>
       <x>715</x>
       <y>15</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-bottom-width: 0px;
	background-color: rgb(0, 170, 255);
	border-radius: 35px;
	font: 12pt &quot;Microsoft YaHei&quot;;
	border-style: outset;
}
QPushButton:hover{
	background-color:rgb(0, 147, 220)
}</string>
     </property>
     <property name="text">
      <string>8</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sys9">
     <property name="geometry">
      <rect>
       <x>815</x>
       <y>15</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-bottom-width: 0px;
	background-color: rgb(0, 170, 255);
	border-radius: 35px;
	font: 12pt &quot;Microsoft YaHei&quot;;
	border-style: outset;
}
QPushButton:hover{
	background-color:rgb(0, 147, 220)
}</string>
     </property>
     <property name="text">
      <string>9</string>
     </property>
    </widget>
    <widget class="QPushButton" name="sys10">
     <property name="geometry">
      <rect>
       <x>915</x>
       <y>15</y>
       <width>70</width>
       <height>70</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="styleSheet">
      <string notr="true">QPushButton{
	border-bottom-width: 0px;
	background-color: rgb(0, 170, 255);
	border-radius: 35px;
	font: 12pt &quot;Microsoft YaHei&quot;;
	border-style: outset;
}
QPushButton:hover{
	background-color:rgb(0, 147, 220)
}</string>
     </property>
     <property name="text">
      <string>10</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tools" native="true">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>300</width>
      <height>100</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-right: 1px solid #dedede;
border-bottom: 1px solid #dedede</string>
    </property>
    <widget class="QPushButton" name="save">
     <property name="geometry">
      <rect>
       <x>125</x>
       <y>25</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Save Results</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="file">
     <property name="geometry">
      <rect>
       <x>25</x>
       <y>25</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Choose Files</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QPushButton" name="clear">
     <property name="geometry">
      <rect>
       <x>225</x>
       <y>25</y>
       <width>50</width>
       <height>50</height>
      </rect>
     </property>
     <property name="cursor">
      <cursorShape>PointingHandCursor</cursorShape>
     </property>
     <property name="toolTip">
      <string>Clear All</string>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="note" native="true">
    <property name="geometry">
     <rect>
      <x>1000</x>
      <y>120</y>
      <width>280</width>
      <height>150</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;

</string>
    </property>
    <widget class="QWidget" name="widget" native="true">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>15</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #E31015;
border-radius: 15px;</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_2" native="true">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>60</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: #64F000;
border-radius: 15px;</string>
     </property>
    </widget>
    <widget class="QWidget" name="widget_3" native="true">
     <property name="geometry">
      <rect>
       <x>15</x>
       <y>105</y>
       <width>30</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(0, 170, 255);
border-radius: 15px;</string>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>15</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Microsoft YaHei&quot;;</string>
     </property>
     <property name="text">
      <string>Root Node</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>60</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Microsoft YaHei&quot;;</string>
     </property>
     <property name="text">
      <string>Node in System</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>105</y>
       <width>200</width>
       <height>30</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 10pt &quot;Microsoft YaHei&quot;;</string>
     </property>
     <property name="text">
      <string>Other System</string>
     </property>
    </widget>
   </widget>
   <widget class="QListWidget" name="resultlist">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>100</y>
      <width>300</width>
      <height>700</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-right: 1px solid #dedede</string>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QWidget" name="loading" native="true">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>1300</width>
      <height>800</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>ArrowCursor</cursorShape>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgba(0, 0, 0, 150);</string>
    </property>
    <widget class="QLabel" name="txtloading">
     <property name="geometry">
      <rect>
       <x>500</x>
       <y>300</y>
       <width>300</width>
       <height>200</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 24pt &quot;Microsoft YaHei&quot;;
background-color: rgba(255, 255, 255, 0);
color: rgb(255, 255, 255);</string>
     </property>
     <property name="text">
      <string>Analyzing...</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="instruction">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>260</width>
      <height>160</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius: 5px;
font: 10pt &quot;Microsoft YaHei&quot;;</string>
    </property>
    <property name="text">
     <string>Tip: To view the error information of the root node, you can move the mouse on the list entry and stay for a while.</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
    <property name="margin">
     <number>0</number>
    </property>
    <property name="indent">
     <number>10</number>
    </property>
   </widget>
   <zorder>systab</zorder>
   <zorder>topologyview</zorder>
   <zorder>tools</zorder>
   <zorder>note</zorder>
   <zorder>resultlist</zorder>
   <zorder>instruction</zorder>
   <zorder>loading</zorder>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
